tests/unit_tests/flow_variable_tests/auxiliary_equations_int_by_parts.cpp

```diff
@@ -0,0 +1,496 @@
+#include <iomanip>
+#include <cmath>
+#include <limits>
+#include <type_traits>
+#include <math.h>
+#include <iostream>
+#include <stdlib.h>
+
+#include <deal.II/distributed/solution_transfer.h>
+
+#include "testing/tests.h"
+
+#include<fstream>
+#include <deal.II/base/parameter_handler.h>
+#include <deal.II/base/tensor.h>
+#include <deal.II/numerics/vector_tools.h>
+
+#include <deal.II/grid/grid_generator.h>
+#include <deal.II/grid/grid_refinement.h>
+#include <deal.II/grid/grid_tools.h>
+#include <deal.II/grid/grid_out.h>
+#include <deal.II/grid/grid_in.h>
+
+#include <deal.II/dofs/dof_handler.h>
+#include <deal.II/dofs/dof_tools.h>
+#include <deal.II/dofs/dof_renumbering.h>
+
+#include <deal.II/dofs/dof_accessor.h>
+
+#include <deal.II/lac/vector.h>
+#include <deal.II/lac/dynamic_sparsity_pattern.h>
+#include <deal.II/lac/sparse_matrix.h>
+
```

Comment:
Why do you need all those headers?

Reply:
You're right, I don't. I copied it from a different file.

Reply:
OK, can you remove those? More header files tend to mean higher build times when dependencies aren't needed.

Reply:
Yes, confirming I removed the unnecessary ones.